src/NinjaTools.FluentMockServer.API/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace NinjaTools.FluentMockServer.API.Extensions
+{
+    public static class StringExtensions
+    {
+        public static bool IsValidRegex(this string pattern)
+        {
+            if (string.IsNullOrEmpty(pattern)) return false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)